src/typeDefs/types.ts

```diff
@@ -347,4 +362,43 @@ export const types = gql`
     edges: [User]!
     aggregate: AggregateUser!
   }
+
+  type UserEdge {
+    node: User
+    cursor: String!
+  }
+
+  type UserTag {
+    _id: ID!
+    name: String!
+    organization: Organization!
```

Comment:
Make this nullable, all fields which make new network requests to get the data should be nullable, you don't want failed network request for a nested field to nullify the whole response object.